tmt/tests/booted/readonly/003-test-container-inspect.nu

```diff
@@ -0,0 +1,15 @@
+use std assert
+use tap.nu
+
+tap begin "verify bootc container inspect"
```

Comment:
This is totally OK, however I think what we really want to validate is that this command works in a container. That's a distinct test suite, see `just test-container` and https://github.com/bootc-dev/bootc/blob/main/crates/tests-integration/src/container.rs